operator/v1/types_machineconfiguration.go

```diff
@@ -41,8 +41,8 @@ type MachineConfigurationSpec struct {
 	// managedBootImages allows configuration for the management of boot images for machine
 	// resources within the cluster. This configuration allows users to select resources that should
 	// be updated to the latest boot images during cluster upgrades, ensuring that new machines
-	// always boot with the current cluster version's boot image. When omitted, no boot images
-	// will be updated.
+	// always boot with the current cluster version's boot image. When omitted, the MCO will
+	// will use cluster defaults, which may vary between releases.
```

Comment:
We have a particular way of wording this
```suggestion
	// When omitted, this means no opinion and the platform is left to choose a reasonable default, which is subject to change over time.
	// The default is ...
```

Reply:
Updated in last push!